vending-app/src/main/java/com/android/vending/licensing/LicenseServiceNotificationRunnable.java

```diff
@@ -70,7 +70,7 @@ public void run() {
         ignoreIntent.putExtra(INTENT_KEY_IGNORE_PACKAGE_NAME, callerPackageName);
         ignoreIntent.putExtra(INTENT_KEY_NOTIFICATION_ID, callerUid);
         PendingIntent ignorePendingIntent = PendingIntent.getBroadcast(
-            context, callerUid * 2 + 1, ignoreIntent, PendingIntent.FLAG_IMMUTABLE
+            context, callerUid * 2 + 1, ignoreIntent, PendingIntent.FLAG_MUTABLE
```

Comment:
I have not one bit of an idea why, in my emulator tests, the `INTENT_KEY_IGNORE_PACKAGE_NAME` extra doesn't reach the receiver when `FLAG_IMMUTABLE` is set.